docs/src/Atmos/Microphysics.md

```diff
@@ -0,0 +1,369 @@
+  - condensation/evaporation of cloud water,
+  - autoconversion,
+  - accretion,
+  - evaporation of rain water.
+
+The cloud microphysics variables are expressed as specific humidities:
+  - q_tot - total water specific humidity,
+  - q_vap - water vapor specific humidity,
+  - q_liq - liquid water specific humidity,
+  - q_rai - rain water specific humidity.
+
+Parameters used in the parameterization are defined in `MicrophysicsParameters` module. They consist of:
+
+|    symbol            |         definition                                        | units                    | default value         |
+|----------------------|-----------------------------------------------------------|--------------------------|-----------------------|
+|``n_{0_{MP}}``        | rain drop size distribution parameter                     | ``\frac{1}{m^4}``        | ``16 \cdot 10^6``     |
+|``\tau_{cond_evap}``  | cloud water condensation/evaporation timescale            | ``s``                    | ``10``                |
+|``\tau_{acnv}``       | cloud to rain water autoconversion timescale              | ``s``                    | ``10^3``              |
+|``q_{liq\_threshold}``| cloud to rain water autoconversion threshold              | -                        | ``5 \cdot 10^{-4}``   |
+|``E_{col}``           | collision efficiency between rain drops and cloud droplets| -                        | ``0.8``               |
+|``C_{drag}``          | rain drop drag coefficient                                | -                        | ``0.55``              |
+|``a_{vent}, b_{vent}``| rain drop ventilation factor coefficients                 | -                        | ``1.5 \;``,``\; 0.53``|
+|``K_{therm}``         | thermal conductivity of air                               | ``\frac{J}{m \; s \; K}``| ``2.4 \cdot 10^{-2}`` |
+|``\nu_{air}``         | kinematic viscosity of air                                | ``\frac{m^2}{s}``        | ``1.6 \cdot 10^{-5}`` |
+|``D_{vapor}``         | diffusivity of water vapor                                | ``\frac{m^2}{s}``        | ``2.26 \cdot 10^{-5}``|
+
+## Rain drop size distribution
+
+The rain-drop size distribution is assumed to follow Marshall-Palmer distribution (Marshall Palmer 1948 eq. 1):
+```math
+\begin{equation}
+n(r) = n_{0_{MP}} exp\left(- \lambda_{MP} \; r \right)
+\end{equation}
+```
+where:
+ - ``r`` is the drop radius,
+ - ``n_{0_{MP}}`` and ``\lambda_{MP}`` are the Marshall-Palmer distribution parameters (twice the values used in the Marshall Palmer 1948, because we use drop radius and not diameter).
+
+## Terminal velocity
+
+The terminal velocity of an individual rain drop is defined by the balance between the gravity (taking into account the density difference between water and air) and the drag force:
+```math
+\begin{equation}
+v_{drop} = \left(\frac{8}{3 \; C_{drag}} \left( \frac{\rho_{water}}{\rho} -1 \right) \right)^{1/2} g^{1/2} \; r^{1/2} = v_c(\rho) \; g^{1/2} \; r^{1/2}
```

Comment:
would be a bit clearer to write this as (g r)^{1/2}

Reply:
done